api/src/main/java/org/openmrs/BaseFormRecordableOpenmrsData.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.api.APIException;
+
+import javax.persistence.Column;
+import javax.persistence.MappedSuperclass;
+
+/**
+ * This is a super class to make a bridge between an OpenMrsObject and is position in the form.
```

Comment:
Just minor comment here on the wording.

'This is a super class to make a bridge between an OpenMrsObject and **is** position in the form.' **->** 'This is a super class to make a bridge between an OpenMrsObject and **its** position in the form.'

Reply:
@icrc-loliveira this maybe?
>`Base implementation of FormRecordable that bridges between a saved BaseChangeableOpenmrsData entity and the path in a form where it was recorded.`